idm/ldif/demousers.ldif

```diff
@@ -0,0 +1,162 @@
+dn: uid=einstein,ou=users,o=libregraph-idm
+objectClass: inetOrgPerson
+objectClass: organizationalPerson
+objectClass: ownCloud
+objectClass: person
+objectClass: top
+uid: einstein
+givenName: Albert
+sn: Einstein
+cn: einstein
+displayName: Albert Einstein
+description: A German-born theoretical physicist who developed the theory of relativity, one of the two pillars of modern physics (alongside quantum mechanics).
+mail: einstein@example.org
+ownCloudUUID:: NGM1MTBhZGEtYzg2Yi00ODE1LTg4MjAtNDJjZGY4MmMzZDUx
```

Comment:
why do we encode the UUID?

Reply:
Hm, this was just copied from our sample LDIFs. But you're right the base64 encoding is not needed here. (LDIF only needs this for values containing non-printable characters, binary data or to avoid escaping. None of those is the case here)

Fixed